scipy/sparse/csgraph/_laplacian.py

```diff
@@ -271,6 +298,57 @@ def laplacian(
            [-1.,  2., -1.],
            [-1., -1.,  2.]])
 
+    The signed graph, where edge weights can cancel each other out:
+
+    >>> G = np.array([[0, 1, -1],
+    ...               [1, 0,  1],
+    ...               [-1, 1, 0]])
+    >>> G
+    array([[ 0,  1, -1],
+           [ 1,  0,  1],
+           [-1,  1,  0]])
+
+    Setting variant="repelling" (default),
+    positive and negative edge weights cancel during degree computation:
+
+    >>> L_repelling, d_repelling = csgraph.laplacian(G, return_diag=True)
+    >>> L_repelling
+    array([[ 0, -1,  1],
+           [-1,  2, -1],
+           [ 1, -1,  0]])
+    >>> d_repelling
+    array([0, 2, 0])
+
+    Setting variant="opposing" prevents cancellation by summing absolute edge weights:
+
+    >>> L_opposing, d_opposing = csgraph.laplacian(
+    ...     G,
+    ...     return_diag=True,
+    ...     variant="opposing",
+    ... )
+    >>> L_opposing
+    array([[ 2, -1, 1],
+           [-1,  2, -1],
+           [1, -1,  2]])
+    >>> d_opposing
+    array([2, 2, 2])
+
+    Setting variant="unsigned" discards the sign of edge weights
+    and computes the Laplacian using absolute edge weights
```

Comment:
It's not clear in this sentence what "computes the Laplacian using absolute edge weights" refers to. Knowing the answer I must guess that "computes the Laplacian" refers to handling the adjacency matrix part of the laplacian formula. But computing the laplacian involves both computing degrees and combining the degrees with the adjacency info. Let's either find better words to use, use formulas, or define terms for the degree handling and for the adjacency handling and use them.